### ISSUE TITLE
Service wrongly assumes that an instance has registration closed

### ISSUE THREAD

**Issue by quimgil** [TheKinrar/mastodon-instances#54]
https://instances.social/red.confederac.io instance log assumes that our instance is closed to registrations but it is not.

```
[Tue, 17 Apr 2018 14:21:00 GMT] [ERROR] Error in post-update tasks: "ResourceRequest timed out".
[Tue, 17 Apr 2018 14:20:50 GMT] [ERROR] Could not detect registrations status: "ETIMEDOUT". Defaulting to closed.
```

Maybe a side effect of using a subdomain, confusing your API calls? The instance itself seems to be working just fine.

PS: Thank you very much for this useful service!

**Comment by TheKinrar** [comment]
This is a known bug. Don't worry about the log, your instance is still considered open by the instances list, and so is listed on joinmastodon.org and the wizard results for example. :)

**Comment by quimgil** [comment]
Alright, thank you very much for your quick reply. 

Is this new warning a consequence of the same bug? Where can I learn about this Observatory and its score? A quick search only returned this error message in other instances.  :)

```
[Tue, 17 Apr 2018 15:15:12 GMT] [WARNING] No cached Observatory score. Score is set to 0.
[Tue, 17 Apr 2018 15:15:12 GMT] [WARNING] Could not get instance Observatory score: "Test isn't finished".
```

**Comment by quimgil** [comment]
The Observatory warning keeps appearing every 25 hours. Can it be that this might be the cause why our instance hasn't showed up yet in joinmastodon.org? I guess it makes it being considered not stable?

**Comment by Cassolotl** [comment]
Related to */list/old says ALL instances are closed to new registrations #82* maybe?

**Comment by TheKinrar** [comment]
Probably same issue as (or related to) #81
